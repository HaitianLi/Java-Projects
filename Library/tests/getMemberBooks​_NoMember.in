member books 100000
exit