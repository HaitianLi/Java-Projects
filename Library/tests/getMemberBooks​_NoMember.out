user: No members in system.

user: Ending Library process.
